plugins/rtmp-services/data/services.json

```diff
@@ -2011,6 +2011,22 @@
                 "max audio bitrate": 256,
                 "max video bitrate": 20480
             }
+        },
+        {
+            "name": "SHOWROOM",
+            "servers": [
+                {
+                    "name": "Default",
+                    "url": "https://www.showroom-live.com/api/obs/streaming_info?obs_key="
+                }
+            ],
+            "recommended": {
+                "keyint": 2,  
+                "profile": "main",
```

Comment:
Are you sure you want to restrict the profile? Main profile is only needed for compatibility with very old devices (10+ years old at this point).

Reply:
To expound on what R1CH is saying here, OBS will force these settings on users' streams by default (except for the bitrate settings, those are treated as a maximum) unless they are in Advanced output mode and uncheck the "Enforce streaming service encoder settings" check box.

Reply:
The reason why the profile is main is because of user traffic and infrastructure costs of our service

Reply:
The user traffic part of that doesn't make any sense to me, the "high" profile just enables additional encoder features that can improve efficiency, if anything it allows you to *reduce* traffic. The only cost I can see for infrastructure is a (probably negligible) increase in decoding complexity for transcoding.

Reply:
Depending on the distribution streaming server SDK we are using and the range of operating systems supported by the client app, we need to select the main profile.